docs/quick_start.md

```diff
@@ -3,9 +3,83 @@ id: quick_start
 title: Quick Start
 ---
 
-To start a new task locally, in development mode follow these steps:
+Before we can get started with Honeycomb, we must make sure our machines have the required prerequisites for Honeycomb to run. 
+### 0. Electron Dependencies and How To Install
+Listed below are the 4 main software that are required across all operating systems to run Honeycomb. Also included are instructions specific to the operating system the user may be using. 
+
+#### Git
+- Git is a version control system. 
+- It allows us to track the changes and updates being made to a project. We are able to see who made which changes. 
+- Git also provides safety mechanisms to ensure collaborative work is done systematically and allow a large group of developers to work on one project together without getting in each others way. 
+
+#### Node.js 
+- Open-source JavaScript platform that allows users to build network applications quickly. It is just another way to execute code on your computer. 
+- Node.js is a popular choice as it is cross-platform, so a Node.js app can run on all major operating systems. 
+- Electron, one of the other main dependencies of the Honeycomb app, requires the Node.js runtime environment.
+
+#### Electron 
+- Framework for building desktop applications using JavaScript, HTML, and CSS 
+- Requires you to have one JavaScript codebase that will create cross-platform apps that work on Windows, MacOS, and Linux.
+- Honeycomb relies on Electron to package the cross-platform desktop application. 
+
+
+##### Prerequisties Specific to Windows
+
+- Visual Studio:
+Install the latest version of [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the Desktop Development for C++ Workflow. 
+To add the workflow, follow [these instructions](https://docs.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-160#:~:text=If%20you%20have%20Visual%20Studio,Then%2C%20choose%20Modify).
```

Comment:
I believe this is only a prerequisite for building local Windows applications? I wonder if we should leave it somewhere else in the docs that talks about building for Windows specifically